.gitignore

```diff
@@ -60,3 +60,7 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+# Cloud9 Junk
+
+.~c9
```

Comment:
I think you have to put `.~c9*` to match any file that starts with '.~c9'.

Before running `git add` on cloud9, if you run `git status`, git will print a list of all files it recognizes as being modified that are candidates for staging.  This has saved me a couple times when I've forgotten to add something to .gitignore or typed it in wrong.